### DISCUSSION TITLE
Queue 란?

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#48] [Q&A] [answered]
Queue 란?

**Comment by tjdud22** [comment, marked as answer]
큐(Queue)는 한쪽 끝에서만 삽입이 이루어지고, 다른 한쪽 끝에서는 삭제 연산만 이루어지는 유한 순서 리스트입니다.
데이터의 입력과 출력 순서는 선입선출(FIFO:First in First Out)입니다.
